pandamon

```diff
@@ -263,12 +316,17 @@ def run():
         elif args.rnge[-1] == '-':
             datasets = [ds for ds in datasets if int(ds['reqid']) >= int(r[0])]
         else:
-            datasets = [ds for ds in datasets if int(ds['reqid']) in range(int(r[0]),int(r[1]))]
+            datasets = [
+                ds
+                for ds in datasets
+                if int(ds['reqid']) in range(int(r[0]), int(r[1]))
+            ]
```

Comment:
This one is also a bit awkward, but I guess this is Black's way of telling me that I should just write a `for` loop.

Reply:
I'm fine with it. This is too long a list comprehension anyway.